### PR TITLE
Orcid empty state

### PR DESCRIPTION
**Pull request by kmaid**
https://trello.com/c/Edup6vrT/1224-orcid-empty-state

### CODE REVIEW

File: apps/storybook/src/UserProfileRecentWorks.stories.tsx [hunk 1]
```diff
@@ -1,34 +1,44 @@
...
-        title:
-          'Recognizing the importance of new tools and resources for research',
-        type: 'WEBSITE',
-        publicationDate: {
-          year: '2015',
-          month: '3',
-        },
-        lastModifiedDate: '1478865224685',
+    orcid={text('Orcid Id', '0000-0002-9945-3696')}
+    orcidWorks={Array.from({ length: number('Number of orcid works', 2) }).map(
```
Comment (jeysal) [inline apps/storybook/src/UserProfileRecentWorks.stories.tsx:13]:
```suggestion
    orcidWorks={Array.from({ length: number('Number of ORCID works', 2) }).map(
```
This is not a Twotter shitpost Tom! :D

File: apps/storybook/src/UserProfileRecentWorks.stories.tsx [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+      (_, i) => {
+        if (i === 0) {
+          return {
+            doi: text('Work Doi', 'https://doi.org/10.7554/elife.07083'),
```
Comment (jeysal) [inline apps/storybook/src/UserProfileRecentWorks.stories.tsx:17]:
```suggestion
            doi: text('Work DOI', 'https://doi.org/10.7554/elife.07083'),
```

File: apps/storybook/src/UserProfileRecentWorks.stories.tsx [hunk 3]
```diff
@@ -1,34 +1,44 @@
...
-        doi: 'https://doi.org/10.7554/elife.07083',
-        title:
-          'Recognizing the importance of new tools and resources for research',
-        type: 'WEBSITE',
-        publicationDate: {
-          year: '2015',
-          month: '3',
-        },
-        lastModifiedDate: '1478865224685',
+    orcid={text('Orcid Id', '0000-0002-9945-3696')}
```
Comment (jeysal) [inline apps/storybook/src/UserProfileRecentWorks.stories.tsx:12]:
```suggestion
    orcid={text('ORCID', '0000-0002-9945-3696')}
```

File: packages/react-components/src/organisms/UserProfileRecentWorks.tsx [hunk 4]
```diff
@@ -129,26 +136,58 @@ const UserProfileRecentWork: React.FC<UserProfileRecentWorkProps> = ({
...
+        <span css={{ display: 'grid', svg: { fill: lead.rgb } }}>
+          {orcidIcon}
+        </span>
+      </div>
+      {orcidWorks.length === 0 && isOwnProfile ? (
+        <Paragraph accent="lead">
+          <span css={titleStyle}>No works available on your ORCID.</span>
+          <br />
+          To complete this section, please add works to your ORCID.{' '}
+          <span css={{ svg: { stroke: fern.rgb } }}>
```
Comment (jeysal) [inline packages/react-components/src/organisms/UserProfileRecentWorks.tsx:159]: move lower. then I think this whole span seems unnecessary

File: packages/react-components/src/organisms/UserProfileRecentWorks.tsx [hunk 5]
```diff
@@ -129,26 +136,58 @@ const UserProfileRecentWork: React.FC<UserProfileRecentWorkProps> = ({
...
-        ])
-        .slice(1)}
-    </ul>
-  </Card>
-);
+  orcid,
+}) => {
+  const { isOwnProfile } = useContext(UserProfileContext);
+
+  return orcidWorks.length || isOwnProfile ? (
```
Comment (jeysal) [inline packages/react-components/src/organisms/UserProfileRecentWorks.tsx:143]: Maybe make the `return null` an early return. Feel like the nested ternary is getting confusing

File: packages/react-components/src/organisms/UserProfileRecentWorks.tsx [hunk 6]
```diff
@@ -129,26 +136,58 @@ const UserProfileRecentWork: React.FC<UserProfileRecentWorkProps> = ({
...
+            </Link>
+          </span>
+          <br />
+          <br />
+          <span css={titleStyle}>Your ORCID is: </span>
+          <Link href={new URL(`https://orcid.org/${orcid}`).toString()}>
+            {orcid}
+          </Link>
+          <br /> If this is incorrect please{' '}
+          <Link href={mailToSupport({ subject: 'Orcid change request' })}>
```
Comment (jeysal) [inline packages/react-components/src/organisms/UserProfileRecentWorks.tsx:172]:
```suggestion
          <Link href={mailToSupport({ subject: 'ORCID change request' })}>
```
Unless this was specified by requirements

**Review by jeysal — CHANGES_REQUESTED** [review]

File: packages/react-components/src/organisms/UserProfileRecentWorks.tsx (hunk 6 above)
Comment (kmaid) [reply packages/react-components/src/organisms/UserProfileRecentWorks.tsx:172]: It was copy pasted from the trello

**Review by jeysal — APPROVED** [review]

**Pull request merged**